husky_description/urdf/dual_arm_husky.urdf.xacro

```diff
@@ -98,9 +98,9 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <!-- Interial link stores the robot's inertial information -->
   <link name="inertial_link">
     <inertial>
-      <mass value="33.455"/>
-      <origin xyz="-0.08748 -0.00085 0.09947"/>
-      <inertia ixx="0.6022" ixy="-0.02364" ixz="-0.1197" iyy="1.7386" iyz="-0.001544" izz="2.0296"/>
+      <mass value="48.39054"/>
+      <origin xyz="-0.0648 -0.00059 0.07895"/>
+      <inertia ixx="2.1634275613800003" ixy="-0.055452352110000006" ixz="-0.41083822026" iyy="3.5314689495400002" iyz="-0.004366145530000001" izz="4.4759585875" />
```

Comment:
We've contacted Clearpath Support and Martin Cote sent us these values. The axes/frame were wrong and I've changed them by inspecting the URDF. It's much better, but the Husky still doesn't drive in Gazebo and is highly unstable. Maybe they are off? I'd be much obliged if you could take a look at it/from the CAD files and figure out whether other values should be put in here